src/v2/api/dtos/VerificationEmailDTO.ts

```diff
@@ -3,7 +3,9 @@ import { validationOptionsMsg } from '../../utils/GLOBALS';
 import { ApiProperty } from '@nestjs/swagger';
 
 export class VerificationEmailDTO {
-  @ApiProperty()
+  @ApiProperty({
+    description: 'An email of the user for whom resending a letter with confirmation of email address is requested',
+  })
   @IsEmail({}, validationOptionsMsg('Email is not email'))
     email: string;
```

Comment:
Add IsNotEmpty decorator for email and appropriate exception
![image](https://github.com/fictadvisor/fictadvisor-api/assets/135731067/2edfc266-ac10-4d70-95ac-c4ee7d71fbbf)